utils/oscap-xccdf.c

```diff
@@ -1172,6 +1225,7 @@ bool getopt_xccdf(int argc, char **argv, struct oscap_action *action)
 		{"enforce-signature", no_argument, &action->enforce_signature, 1},
 		{"fetch-remote-resources", no_argument, &action->remote_resources, 1},
 		{"progress", no_argument, &action->progress, 1},
+		{"progress-full", no_argument, &action->progress, 2},
```

Comment:
Consider replacing magic constants 1 and 2 by an enum.

Reply:
Done.